src/hooks/useColorGenerator/types.ts

```diff
@@ -0,0 +1,29 @@
+export type ColorOptions = {
+    lightness: [number, number];
+    situration: [number, number];
+};
+
+export type ThemeColorSettings = {
+    light: ColorOptions;
+    medium: ColorOptions;
+    heavy: ColorOptions;
+};
+
+export type INTENSITY = 'light' | 'medium' | 'heavy';
```

Comment:
Types are always PascalCase
```suggestion
export type Intensity = 'light' | 'medium' | 'heavy';
```